src/context/WalletProvider/NewWalletViews/NewWalletViewsSwitch.tsx

```diff
@@ -38,18 +53,87 @@ type RightPanelContentProps = {
   setIsLoading: (loading: boolean) => void
   error: string | null
   setError: (error: string | null) => void
+  location: Location
 }
 
+const nativeRoutes = (
```

Comment:
This will get ugly pretty fast, but endgame here is not needing this at all after new flow go-live, see https://github.com/shapeshift/web/issues/8596